packages/codegen-ui-react/lib/__tests__/__utils__/amplify-renderer-generator.ts

```diff
@@ -156,3 +156,11 @@ export const renderExpanderJsxElement = (
 
   return transpile(expanderNode, {}).componentText;
 };
+
+export const rendererConfigWithGraphQL: ReactRenderConfig = {
```

Comment:
I think it might be better to take `dataApi` one level down so we contain everything related to that. 

```
{
  dataApi: 'GraphQL',
  operationsFilePath: '../graphql',
  typesFilePath: '../API',
}
```

or 

```
{
  dataApi: 'DataStore'
}
```

Reply:
updated per suggestion